.vsts-ci.yml

```diff
@@ -40,13 +40,6 @@ resources:
         externals: true
         tools: true
         tasks: true
-    - container: rbase
-      image: wch1/r-debug
-      mountReadOnly:
-        work: false
-        externals: true
-        tools: true
-        tasks: true
```

Comment:
Just a small bit of unrelated cleanup, since I'm touching this file anyway.

We don't need this R image in the Azure DevOps config any more, since #7032